compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/arguments/CommonCompilerArguments.kt

```diff
@@ -415,6 +415,9 @@ abstract class CommonCompilerArguments : CommonToolArguments() {
     @Argument(value = "-Xrender-internal-diagnostic-names", description = "Render internal names of warnings and errors")
     var renderInternalDiagnosticNames: Boolean by FreezableVar(false)
 
+    @Argument(value = "-Xenable-assign-operator-for-f10", description = "Enable assign operator overload for frontend 1.0")
```

Comment:
Just `-Xassign-operator`

Reply:
It's also required to put all your changes under a feature for FIR too

Reply:
I planned to have different flag for old and new frontend. That is because for old frontend, I have to handle it in the backend. But maybe I can just check that if it's K2 compiler, I don't enable backend handling.

What are your thoughts?

Reply:
I meant psi2ir yes. Aaa, I didn't know that, I though that that code always run if IR backend is enabled. Thx, that simplifes, I will change it to `-Xassign-operator`.

Reply:
FYI: for FIR we have similar part which is called fir2ir